library/aes.c

```diff
@@ -1918,8 +1910,9 @@ int mbedtls_aes_self_test(int verbose)
 
             /*
              * AES-192 is an optional feature that may be unavailable when
-             * there is an alternative underlying implementation i.e. when
-             * MBEDTLS_AES_ALT is defined.
+             * there is an alternative underlying implementation i.e when
```

Comment:
Please change the three other instances as well, lines 1978-1979 ...

Reply:
Whooops... sorry...